code/game/atoms.dm

```diff
@@ -1384,7 +1384,17 @@
 /atom/proc/log_message(message, message_type, color=null, log_globally=TRUE)
 	if(!log_globally)
 		return
-
+	if(CONFIG_GET(flag/sql_game_log) && CONFIG_GET(flag/sql_enabled))
+		//Build SQL Query to insert log into the DB, and catch errors
+		var/datum/db_query/query_sql_log_messages = SSdbcore.NewQuery({"
```

Comment:
I would express concerns about the performance of doing an INSERT every single time a game log line is produced.

Reply:
Testing it on my end, it slightly improved performance due to less Disk IOPS from writing to filesystem.

Reply:
Ensure to test with appropriate network latency, the database is not local to the servers in many cases.